src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -23,8 +23,8 @@ class CustomizedAxisTick extends React.Component {
 
 function getDates(startDate, endDate, days) {
   const dates = [];
-  let currentDate = new Date(startDate);
-  const endingDate = new Date(endDate);
+  let currentDate = new Date(startDate).replace(/-/g, '/');
+  const endingDate = new Date(endDate).replace(/-/g, '/');
```

Comment:
since we are importing `moment`, should we use it throughout this code to parsing dates from strings?

basically we would format our date strings in to ISO 8601 formats and call `moment()` to parse it https://momentjs.com/docs/#/parsing/string/ this way we could avoid calling native `new Date()` and got unexpected results in the future

Reply:
valid point 👍  @mfshao